Cargo.toml

```diff
@@ -29,7 +29,7 @@ http-body = "0.3.1"
 httparse = "1.0"
 h2 = "0.2.2"
 itoa = "0.4.1"
-tracing = { version = "0.1", default-features = false, features = ["std", "log"] }
+tracing = { version = "0.1", features = ["log"] }
```

Comment:
if we're not using attributes, i think it's better to leave it off --- this means that if users opt out of it, it will still be pulled.

Reply:
What do you mean? Doesn't it need to be kept so that users who were already using some `log`ger will still get logs?

Reply:
fixed!

Reply:
they'll keep getting logs, but they won't get https://docs.rs/tracing/0.1.15/tracing/attr.instrument.html.

Reply:
Or rather, _Hyper_ won't be pulling in that macro.